db/migrations/20160808182741_isolation_segment.rb

```diff
@@ -0,0 +1,9 @@
+Sequel.migration do
+  change do
+    create_table :isolation_segments do
+      VCAP::Migration.common(self)
+      String :name, null: false, case_insensitive: :true, unique: true
```

Comment:
Can you name this uniqueness constraint so that it is the same in mysql and psql?